Umbraco-Headless/Getting-Started-Cloud/The-Headless-Portal/index.md

```diff
@@ -0,0 +1,71 @@
+# The Umbraco Heartcore Portal
+
+After you have logged into the Cloud Portal you are presented with the project overview.
+
+![Portal Project Overview](images/portalOverview.png)
+
+From here you can see a list of all your projects. If you have many projects, you are able to search for specific projects from the search bar in the top right corner.
+
+Clicking on a project will take you to the project where you will get an overview of your environment(s).
+
+![Portal Project Overview](images/projectOverview.png)
+
+## Manage Environments
+
+From here you are able to create multiple environment so you might have a Development environment as well. Depending on what plan you have chosen, you will have acces to one or more environment on your project.
+
+![Manage the environments](images/manageEnvironments.png)
+
+If you want to add an environment all you have to do is click the big green button. Umbraco will create the environment for you.
+
+When you are done you can go a head and click on the Done, take me back link in the top-right corner to get back to the project overview.
+
+## Invite member
+
+In the top-right corner you will find the **invite member** option. This will allow you to quickly invite new team members to your project from a modal that pops up. This makes it very fast and easy to invite new team members. Depending on your chosen plan, you might be charged an extra €5 euros per added team member.
+
+![Invite Team Member Modal](images/inviteModal.png)
+
+## Settings
+
+Right next to the invite member option. you will find the **Settings** dropdown menu. Here you will find various links to more management and configuration options.
+
+### Edit Team
+
+As with the **invite member** option you are able to add new team members to your project. You will also be able to remove team members from this page as well as see any pending invitations you might have sent.
+
+It is also here you are able to add the Technical Contact information such as Name, Email and a telephone number.
+
+![Edit Team Members page](images/editTeam.png)
+
+### Webhooks
+
+Here you are able to set up a webhook that will gather all the information related to deployments. This information is very useful if you har set up something like Slack integration so every time a deployment has been made a message will be posted.
+
+The webhooks that can be added here is not the same as the webhooks that can be created from the Backoffice.
```

Comment:
Would be cool if this was added like a note or tip instead, right?

Check out this article to learn more about how to format notes and tips: https://our.umbraco.com/documentation/Contribute/Markdown-Conventions/#adding-notes-warnings-tips